recipes-bsp/sgx/sgx_2.12.bb

```diff
@@ -29,25 +29,50 @@ do_compile_prepend () {
 
 do_install () {
     # Package PSW installer
-    install -d "${D}${sgxdirprefix}${sgxrootdir}"
-    install -m 0755 "${B}/linux/installer/bin/sgx_linux_x64_psw_2.12.100.3.bin" "${D}${sgxdirprefix}${sgxrootdir}"
+    install -d "${D}${sgxdirprefix}${sgxrootdir}/package"
+    install -m 0755 "${RECIPE_SYSROOT}${sgxrootdir}/package/sgx_linux_x64_sdk_2.12.100.3.bin" "${D}${sgxdirprefix}${sgxrootdir}/package"
+    install -m 0755 "${B}/linux/installer/bin/sgx_linux_x64_psw_2.12.100.3.bin" "${D}${sgxdirprefix}${sgxrootdir}/package"
+
+    # Package PSW installer
 }
 
 ### package ###
 
 SYSROOT_DIRS += "${sgxrootdir}"
-FILES_${PN}  += "${sgxrootdir}/sgx_linux_x64_psw_2.12.100.3.bin"
+FILES_${PN}  += "${sgxrootdir}"
 
+# TODO: Fix script or implement install on first boot.
+# This script causes fist boot to hang.
 pkg_postinst_ontarget_${PN} () {
-    "${sgxrootdir}/sgx_linux_x64_psw_2.12.100.3.bin"
-    rm -f "${sgxrootdir}/sgx_linux_x64_psw_2.12.100.3.bin"
+    # Verify on target
+    if [ -z "$D" ]; then
+
+        # Install SDK (optional)
+        if [ -e "${sgxrootdir}/package/sgx_linux_x64_sdk_2.12.100.3.bin" ]; then
+            "${sgxrootdir}/package/sgx_linux_x64_sdk_2.12.100.3.bin" --prefix /opt/intel < /dev/null
```

Comment:
is this /opt/intel referring here is on yocto rootfs image ? just want to make sure its not causing harm to host system.

Reply:
https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#migration-2.6-post-installation-changes this seems to be the case, though I am concerned what happens on a read-only rootfs.